### DISCUSSION TITLE
How to specify different ssh port?

### DISCUSSION THREAD

**Discussion by mxwmnn** [borgbase/vorta#1450] [FAQ] [answered]
Cause of the xxx@xxx.com:repo notation how can i specifiy a ssh port different from 22?

**Comment by real-yfprojects** [comment, marked as answer]
You can use this syntax: `ssh://user@host:port/path/to/repo`. See [the borg docs](https://borgbackup.readthedocs.io/en/stable/usage/general.html#repository-urls).
